### PR TITLE
updated cluster model and made drop down more in style

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

**Comment by edewit** [comment]
@invincibleJai please review

File: src/app/launcher/create-app/targetenvironment-createapp-step/target-environment-createapp-step.component.html [hunk 1]
```diff
@@ -31,9 +31,21 @@ <h2 class="card-pf-title--title">{{targetEnv.header}}</h2>
...
-                <option *ngFor="let cluster of targetEnv.clusters">{{cluster.id}}</option>
-              </select>
+
+              <div class="dropdown text-center" dropdown *ngIf="targetEnv.clusters">
+                <button type="button" class="btn btn-default dropdown-toggle" dropdownToggle>
+                  <span *ngIf="!launcherComponent.summary.cluster">Select cluster</span>
+                  <span *ngIf="launcherComponent.summary.cluster">{{launcherComponent.summary.cluster.id}}</span>
+                  <span class="caret"></span>
+                </button>
+                <ul class="dropdown-menu" role="menu" *dropdownMenu style="left: 50%;right: auto;transform: translate(-50%, 0);"
```
Comment (invincibleJai) [inline src/app/launcher/create-app/targetenvironment-createapp-step/target-environment-createapp-step.component.html]: inline styles we can have in `target-environment-createapp-step.component.less`

Comment (invincibleJai) [reply src/app/launcher/create-app/targetenvironment-createapp-step/target-environment-createapp-step.component.html]: @edewit we can handle it in next PR

**Comment by edewit** [comment]
right changed to use .less class

**Pull request merged**